ci_framework/roles/build_containers/tasks/install.yml

```diff
@@ -11,17 +11,26 @@
       - buildah
     state: latest  # noqa: package-latest
 
-- name: Get tcib from repository  # noqa: latest[git]
-  ansible.builtin.git:
-    accept_hostkey: true
-    dest: "{{ cifmw_build_containers_basedir }}/tmp/tcib"
-    repo: "{{ cifmw_build_containers_tcib_src }}"
+- name: Install tcib from source
+  when: cifmw_build_containers_install_from_source | bool
+  block:
+    - name: Get tcib from repository  # noqa: latest[git]
+      ansible.builtin.git:
+        accept_hostkey: true
+        dest: "{{ cifmw_build_containers_basedir }}/tmp/tcib"
+        repo: "{{ cifmw_build_containers_tcib_src }}"
 
-  # FIXME(Chandan/arxcruz): Skip usage of sudo installation
-- name: Install tcib package
+      # FIXME(Chandan/arxcruz): Skip usage of sudo installation
+    - name: Install tcib package
+      become: true
+      ansible.builtin.shell: |
+        pip install -r requirements.txt
+        python setup.py install
+      args:
+        chdir: "{{ cifmw_build_containers_basedir }}/tmp/tcib"
+
+- name: Install tcib from RPM
+  when: not cifmw_build_containers_install_from_source | bool
   become: true
-  ansible.builtin.shell: |
-    pip install -r requirements.txt
-    python setup.py install
-  args:
-    chdir: "{{ cifmw_build_containers_basedir }}/tmp/tcib"
+  ansible.builtin.package:
+    name: python3-tcib
```

Comment:
in fact, this ^ is the new thing here, installing python3-tcib from package.
we were already installing from source, looking at the diff right?

Reply:
If we want to install tcib from source, we need to set `cifmw_build_containers_install_from_source` to true now.